cmd/token/token_test.go

```diff
@@ -410,3 +410,81 @@ func generateJWK(t *testing.T) (*rsa.PrivateKey, jwk.Key) {
 
 	return privateKey, key
 }
+
+func TestCreateInternalJWT(t *testing.T) {
+	// TODO: Although the secret here is not really valuable, it's better to generate a valid temporary one later on
```

Comment:
I agree. In fact, you can just create a config and dump it into the []byte instead of having the text here.